modules/client/src/interfaces.ts

```diff
@@ -309,16 +309,18 @@ export enum DaoSortBy {
 }
 
 export enum SubgraphPluginTypeName {
-  ERC20 = "ERC20VotingPlugin",
-  ADDRESS_LIST = "AddresslistPlugin",
+  TOKEN_VOTING = "TokenVotingPlugin",
+  ADDRESS_LIST = "AddresslistVotingPlugin",
+  ADMIN = "AdminPlugin",
 }
 
 export const SubgraphPluginTypeMap: Map<
   SubgraphPluginTypeName,
   string
 > = new Map([
-  [SubgraphPluginTypeName.ERC20, "erc20voting.dao.eth"],
+  [SubgraphPluginTypeName.TOKEN_VOTING, "tokenvoting.dao.eth"],
```

Comment:
There's another place where this is called `token-voting-...`. Let's unify these domains, please.

Additionally, `dao.eth` is used for DAO's, not for plugins

Reply:
sure, let me know whats the final domain `token-voting.plugin.dao.eth`?

Reply:
I don't think there's a final domain as of yet for plugins. Can we encapsulate this somewhere?  My original point was that I'd swear I had seen this elsewhere in a different way, hence the comment.

Reply:
indeed, it was token-voting.plugin.dao.eth in the examples